cou/utils/nova_compute.py

```diff
@@ -85,3 +94,15 @@ async def verify_empty_hypervisor(unit: Unit, model: Model) -> None:
             unit_instance_count,
         )
         raise HaltUpgradeExecution(f"Unit: {unit.name} has {unit_instance_count} VMs running")
+
+
+def stringify_units(units: Iterable[Unit]) -> str:
+    """Convert Units into a comma-separatedstring of unit names, sorted alphabetically.
+
+    :param units: A iterable of Unit objects to be converted.
+    :type units: Iterable[Unit]
+    :return: A comma-separated string of sorted unit names.
+    :rtype: str
+    """
+    sorted_unit_names = sorted([unit.name for unit in units], key=lambda name: name)
```

Comment:
The key is unnecessary here, e.g. `sorted([unit.name for unit in units], key=lambda name: name)` is the same as `sorted([unit.name for unit in units])`